daemon/daemon.go

```diff
@@ -193,9 +193,11 @@ func (daemon *Daemon) Register(container *Container) error {
 
 	container.daemon = daemon
 
-	// Attach to stdout and stderr
-	container.stderr = broadcastwriter.New()
-	container.stdout = broadcastwriter.New()
+	if container.getLogConfig().Type != "none" {
+		// Attach to stdout and stderr
+		container.stderr = broadcastwriter.New()
```

Comment:
I want to redirect all the logs to /dev/null and I can't do that right now. Should I add another option `--log-driver=null` which redirect logs to `/dev/null`? 
I want to do this cause the oom issue of docker such as https://github.com/docker/docker/issues/14460
Or another option like `--log-driver=naive` which redirects logs to a simple file without passing through docker daemon?  
WDYT ?